tests/Deserializer/Organizer/WebsiteJSONDeserializerTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace CultuurNet\UDB3\Symfony\Deserializer\Organizer;
+
+use CultuurNet\Deserializer\MissingValueException;
+use ValueObjects\StringLiteral\StringLiteral;
+use ValueObjects\Web\Url;
+
+class WebsiteJSONDeserializerTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var WebsiteJSONDeserializer
+     */
+    private $websiteJSONDeserializer;
+
+    protected function setUp()
+    {
+        $this->websiteJSONDeserializer = new WebsiteJSONDeserializer();
+    }
+
+    /**
+     * @test
+     */
+    public function it_can_serialize_a_valid_website()
+    {
+        $json = new StringLiteral('{"website":"http://www.depot.be"}');
+
+        $actual = $this->websiteJSONDeserializer->deserialize($json);
+
+        $this->assertEquals(
+            Url::fromNative('http://www.depot.be'),
+            $actual
+        );
+    }
+
+    /**
+     * @test
+     */
+    public function it_throws_an_exception_when_website_is_missing()
+    {
+        $json = new StringLiteral('{"url":"http://www.depot.be"}');
```

Comment:
Als we in Swagger docs "website" vervangen met "url" dan zou ik dit ook wel doortrekken naar de JSON property, dus dan zou "url" geldig moeten zijn en "website" niet.